src/Castle.Core/DynamicProxy/IProxyGenerator.cs

```diff
@@ -231,7 +233,6 @@ TInterface CreateInterfaceProxyWithTarget<TInterface>(TInterface target, params
 		///   Object proxying calls to members of <paramref name = "interfaceToProxy" /> type on <paramref name = "target" /> object or alternative implementation swapped at runtime by an interceptor.
 		/// </returns>
 		/// <exception cref = "ArgumentNullException">Thrown when given <paramref name = "interfaceToProxy" /> object is a null reference (Nothing in Visual Basic).</exception>
-		/// <exception cref = "ArgumentNullException">Thrown when given <paramref name = "target" /> object is a null reference (Nothing in Visual Basic).</exception>
```

Comment:
Removing this XML documentation comment for the whole `CreateInterfaceProxyWithTargetInterface` method group because `target` is in fact optional (i.e. allowed to be `null`).